bloom-and-vine/src/Pages/Contact.js

```diff
@@ -1,11 +1,133 @@
+                  <path d="M4 4H20C21.1 4 22 4.9 22 6V18C22 19.1 21.1 20 20 20H4C2.9 20 2 19.1 2 18V6C2 4.9 2.9 4 4 4Z" 
+                  stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                  <path d="M22 6L12 13L2 6" 
+                  stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                </svg>
+              </div>
+            </a>
+            </li>
+          <li>
+          <a href='https://pinterest.com' target='_blank'>
+              <div className='PContact'>
+                <svg width="40" height="40" viewBox="0 0 18 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+                  <path fill-rule="evenodd" clip-rule="evenodd" d="M7.50211 15.719C8.23021 15.7425 8.88785 16.7055 
+                    10.8138 16.7055C14.4308 16.729 16.9204 13.9575 17.6955 10.317C20.3965 -2.29555 2.00615 -3.30549 
+                    0.127182 7.07578C-0.319072 9.51843 0.409027 12.3369 2.31148 13.2529C3.76767 13.9575 3.86162 11.9376 
+                    3.41537 11.0686C1.44246 7.2167 3.81465 3.74062 6.91494 2.75416C9.82733 1.81468 11.8942 2.58976 13.4678 
+                    4.21036C15.4877 6.30071 14.5717 11.9846 12.176 13.7931C11.3305 14.4272 9.78036 14.5681 8.9818 13.8166C7.33771 
+                    12.2899 9.87431 9.04869 9.49851 6.6765C9.12272 4.3278 5.41177 4.6801 5.22387 8.1327C5.12992 9.89423 5.64664 10.4109 
+                    5.24736 12.1725C4.61321 14.9909 2.68727 20.7452 4.04952 23C6.04592 22.0605 7.03237 16.4236 7.50211 15.719Z" fill="black"/>
+                </svg>
+              
+              </div>
+            </a>
+          </li>
+        </ul>
+          </div>
+        </div>
+
+      </div>
+
+      <div className='Rightside-Screen'>
+        <div >
+          <form className='form-wrapper'>
+          <label for="name">Name *</label>
+          <input type="text" id="name" name="name"></input>
+          <label for="email">Last Name *</label>
+          <input type="text" id="email" name="email"></input>
+          <label for="phone">Phone *</label>
+          <input type="text" id="phone" name="phone"></input>
+          <label for="date">Date of Event</label>
+          <input type="date" id="email" name="email" ></input>
+          {/* <label for="country">Date of Event</label>
+          <select id="country" name="country" placeholder='mm/dd/yyyy'>
+            <option value="n/a"></option>
+            <option value="n/a"></option>
+            <option value="n/a"></option>
+          </select> */}
+
+          <label for="country">Subject</label>
+          <select id="country" name="country">
+            <option value="n/a"></option>
+            <option value="sub1">Subject 1</option>
+            <option value="sub2">Subject 2</option>
+            <option value="sub3">Subject 3</option>
+          </select>
+
+          
+
+          <label for="subject">Message</label>
```

Comment:
change the for, id, and name attribute to their respective titles. This is important for the autofill feature when filling out the form